tests/functional/api/test_bulk.py

```diff
@@ -13,14 +13,22 @@ class TestBulk:
 
     def test_it_requires_authentication(self, app, bad_header):
         response = app.post(
-            "/api/bulk", b"dummy", headers=bad_header, expect_errors=True
+            "/api/bulk",
+            b"dummy",
+            headers=bad_header,
+            expect_errors=True,
+            content_type="application/x-ndjson",
```

Comment:
Adding the content type here is the only change. This was originally blank causing `webtest` to send `application/x-url-encoded` instead, which triggered a bug in `pyramid-sanity`. That's since been fixed, but it's a good thing to do anyway.